### PR TITLE
Establish initial project structure

### LINKED ISSUE #6: Update README
Update README file to reflect the new direction of the project

### PR DESCRIPTION
**Pull request by niksirbi**
* Adds initial project structure - closes issue #1 
* Populates the README file with basic info - closes #6

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,100 @@
+[project]
+name = "movement"
+authors = [
```
Comment (adamltyson) [inline pyproject.toml:3]: I would only list authors as they contribute significantly to the project. I'm all for sharing credit, but these emails are the ones listed on PyPI, and I would expect that person to be one of the core maintainers. Contributions are listed on the GH page anyway.

Comment (niksirbi) [reply pyproject.toml:3]: Undetstood. So for now just list myself and add other later, as required?

Comment (adamltyson) [reply pyproject.toml:3]: Yeah I think so. Anyone who can answer questions about it I think.

File: .github/workflows/test_and_deploy.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      - uses: neuroinformatics-unit/actions/check_manifest@main
+
+  test:
+    needs: [linting, manifest]
+    name: ${{ matrix.os }} py${{ matrix.python-version }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # Run all supported Python versions on linux
+        python-version: ["3.8", "3.9", "3.10"]
```
Comment (adamltyson) [inline .github/workflows/test_and_deploy.yml:30]: It's probably adding 3.11 to this now, unless you envision any dependencies that aren't yet compatible.

File: LICENSE [hunk 3]
```diff
@@ -1,21 +1,28 @@
-MIT License
 
-Copyright (c) 2020 Adam Tyson
+Copyright (c) 2023, Niko Sirmpilatze
```
Comment (adamltyson) [inline LICENSE:2]: Don't worry about changing this, but we should come up with a standard thing to put here, probably UCL.

File: README.md [hunk 4]
```diff
@@ -1,8 +1,71 @@
...
+* [PyRat](https://github.com/pyratlib/pyrat)
+* [Kino](https://github.com/BrancoLab/Kino)
+* [WAZP](https://github.com/SainsburyWellcomeCentre/WAZP)
+
+## How to contribute
+### Setup
+* Clone the repository and install the package in editable mode (including all `dev` dependencies):
+
+  ```bash
+  git clone https://github.com/neuroinformatics-ua/movement
```
Comment (adamltyson) [inline README.md:31]: typo?

File: pyproject.toml [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+    "fancylog",
+]
+
+classifiers = [
+    "Development Status :: 2 - Pre-Alpha",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
```
Comment (adamltyson) [inline pyproject.toml:26]: Same as above re 3.11

**Review by adamltyson — APPROVED** [review]

File: .github/workflows/test_and_deploy.yml (hunk 2 above)
Comment (niksirbi) [reply .github/workflows/test_and_deploy.yml]: Added.

**Pull request merged**